### PR TITLE
newkaliningrad.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by J0WI** [comment]
Is rufox.ru related to this host?

File: src/chrome/content/rules/newkaliningrad.ru.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<ruleset name="newkaliningrad.ru">
+	<target host="newkaliningrad.ru" />
+	<target host="www.newkaliningrad.ru" />
```
Comment (J0WI) [inline src/chrome/content/rules/newkaliningrad.ru.xml]: m.newkaliningrad.ru

**Comment by galeksandrp** [comment]
split rufox.ru into #5627

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**